src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/highlightablebutton/HighlightableButton.kt

```diff
@@ -3,45 +3,42 @@ package org.wycliffeassociates.otter.jvm.app.widgets.highlightablebutton
 import com.jfoenix.controls.JFXButton
 import de.jensd.fx.glyphs.materialicons.MaterialIconView
 import javafx.beans.property.SimpleBooleanProperty
+import javafx.beans.property.SimpleObjectProperty
 import javafx.scene.paint.Color
 import tornadofx.*
 
 class HighlightableButton : JFXButton() {
 
-    var primaryColor: Color = Color.BLACK
+    var isHighlightedProperty: SimpleBooleanProperty = SimpleBooleanProperty(false)
+    var highlightColorProperty = SimpleObjectProperty<Color>(Color.BLACK)
```

Comment:
Renamed this to `highlightColorProperty` so it wouldn't conflict with `primaryColorProperty` in `ResourceCard`